conf/rubocop.yml

```diff
@@ -139,3 +139,20 @@ Style/TrailingCommaInArrayLiteral:
 
 Style/TrailingCommaInHashLiteral:
   EnforcedStyleForMultiline: consistent_comma
+
+### New Cops
```

Comment:
To avoid the meaning of "New" here decaying over time, what do you think about something specifically around them being pending? "Cops pending in v0.81.0" might be too specific, but something along those lines?

Reply:
Modified the comment here with more info and broken by release.